digits/static/js/ngStorage.js

```diff
@@ -0,0 +1,233 @@
+(function (root, factory) {
```

Comment:
Can we use a minimized version of this file and add a copyright notice to the top? You didn't edit it, did you?